internal/server/data/data.go

```diff
@@ -116,11 +120,34 @@ func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error)
 	return result, nil
 }
 
+func setOrg(db *gorm.DB, model any) {
+	member, ok := model.(orgMember)
+	if !ok {
+		return
+	}
+
+	org := OrgFromContext(db.Statement.Context)
+	member.SetOrganizationID(org.ID)
+}
+
+type orgMember interface {
+	IsOrganizationMember()
+	SetOrganizationID(id uid.ID)
+}
+
+func hasOrgID(model any) bool {
```

Comment:
```suggestion
func isOrgMember(model any) bool {
```

Seems like a more appropriate function name